Lesson9/Gokhan/styles.css

```diff
@@ -0,0 +1,104 @@
+
+/* i added text content, pre element and content centeren on page */
+/* when i was write and save */
+/* i can^t use live server for this work */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
+  display:flex;
```

Comment:
Don't make body a flex container, normally most of the elements inside it are block-level elements already that will take entire row.
```suggestion
```